src/DataStructures/Python/ToNumpy.cpp

```diff
@@ -0,0 +1,38 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "DataStructures/Python/ToNumpy.hpp"
+
+#include <array>
+#include <cstddef>
+#include <utility>
+
+#include "DataStructures/Matrix.hpp"
+#include "DataStructures/Python/Numpy.hpp"
+
+namespace py_bindings {
+
+// Convert Matrix to a Numpy Array.  Always creates a copy.
```

Comment:
Move this documentation into the header file and format for Doxygen (the malloc stuff can stay in the implementation I think)